arangod/RestServer/Metric/BatchBuilder.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2021 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Valery Mironov
+////////////////////////////////////////////////////////////////////////////////
+
+#include <RestServer/Metric/Batch.h>
+#include <RestServer/Metric/Builder.h>
+
+namespace arangodb::metric {
+
+template <typename T>
+class BatchBuilder : public GenericBuilder<BatchBuilder<T>> {
+ public:
+  using MetricT = Batch<T>;
+
+  [[nodiscard]] std::string_view type() const noexcept final {
+    return "untyped";
+  }
+  [[nodiscard]] std::shared_ptr<Metric> build() const final {
```

Comment:
Should be marked as `override`.
```suggestion
  [[nodiscard]] std::shared_ptr<Metric> build() const final {
```